lib/Traits/DiscretelyTimeVaryingTraits.ts

```diff
@@ -46,6 +46,13 @@ export default class DiscretelyTimeVaryingTraits extends mixTraits(
   })
   chartDisclaimer?: string;
 
+  @primitiveTrait({
+    type: "string",
+    name: "Chart color",
+    description: "The color to use when the data set is displayed on the chart"
```

Comment:
Are we expecting any type of value here? eg hex string or rgb string? Perhaps we can just make it clearer?

Reply:
Any html color value should work. I'll add that to the description.